src/compiler/types.ts

```diff
@@ -5459,12 +5459,9 @@ export const enum NodeBuilderFlags {
     AllowEmptyTuple                         = 1 << 19,
     AllowUniqueESSymbolType                 = 1 << 20,
     AllowEmptyIndexInfoType                 = 1 << 21,
-    /** @internal */ WriteComputedProps      = 1 << 30, // { [E.A]: 1 }
-    /** @internal */ NoSyntacticPrinter     = 1 << 31,
     // Errors (cont.)
     AllowNodeModulesRelativePaths           = 1 << 26,
-    /** @internal */ DoNotIncludeSymbolChain = 1 << 27,    // Skip looking up and printing an accessible symbol chain
-    /** @internal */ AllowUnresolvedNames = 1 << 32,

```

Comment:
Given `AllowUnresolvedNames` and `NoTruncation` are the same, I think we need to go vet the places we passed in `NoTruncaction` before to see if they require `AllowUnresolvedNames`. That may be the source of the types baseline changes.

Reply:
I think the latest commit fixed this by just adding the `AllowUnresolvedNames` flag it two spots. No tests baselines are changed after this change.

Reply:
Awesome!